src/main/scala/microsites/util/MicrositeHelper.scala

```diff
@@ -26,11 +26,43 @@ import microsites.util.FileHelper._
 import sbt._
 
 import scala.io.Source
+import com.sksamuel.scrimage._
 
 class MicrositeHelper(config: MicrositeSettings) {
   implicitly(config)
 
   val jekyllDir = "jekyll"
+  val faviconSizes = Seq(16,
+                         24,
+                         32,
+                         48,
+                         57,
+                         60,
+                         64,
+                         70,
+                         72,
+                         76,
+                         96,
+                         114,
+                         120,
+                         128,
+                         144,
+                         150,
+                         152,
+                         196,
+                         310).map { w =>
+    (w, w)
+  } ++ Seq((310, 150))
```

Comment:
Thanks for the suggestion! That size is for Windows 8/IE11 tiled favicons apparently.